pkgs/tools/security/sequoia-chameleon-gnupg/default.nix

```
@@ -0,0 +1,33 @@
+{ lib
+, rustPlatform
+, fetchFromGitLab
+, pkg-config
+, nettle
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "sequoia-chameleon-gnupg";
+  version = "0.1.0";
+
+  src = fetchFromGitLab {
+    owner = "sequoia-pgp";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-msJOQUqq0S+cttSucvXxAFIcFu3R4lUCgIpkaumkaZY=";
+  };
+
+  cargoHash = "sha256-rnZPZY4mg18Plwcrppn7a1+H6CI/A4ntQ5TsZvi6kcA=";
+
+  nativeBuildInputs = [ rustPlatform.bindgenHook pkg-config ];
+  buildInputs = [ nettle ];
+
+  # gpgconf: error creating socket directory
+  doCheck = false;
+
+  meta = with lib; {
+    description = "Sequoia's reimplementation of the GnuPG interface";
+    homepage = "https://sequoia-pgp.org/";
```

Maybe be a bit more specific?

```suggestion
    homepage = "https://gitlab.com/sequoia-pgp/sequoia-chameleon-gnupg";
```